npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/version_resolver.rb

```diff
@@ -222,6 +225,33 @@ def updated_monorepo_dependencies
           updates
         end
 
+        def types_update_available?
+          # TODO: This should probably actually check if there is an updated version
+          #      along with just the existence of the dependency.
+          types_name = PackageName.new(dependency.name).types_package_name
+          types_dep_to_update = top_level_dependencies.find { |d| d.name == types_name.to_s }
+          return true if types_dep_to_update
```

Comment:
I think we can take advantage of `Enumerable#any?` for this and remove the explicit return 🙂 What do you think of?
````suggestion
          types_package_name = PackageName.new(dependency.name).types_package_name.to_s
          top_level_dependencies.any? { |d| d.name == types_package_name }
````